Kitty Items Documentation update (#264)
onflow/kitty-items: .ki-scripts/README.md

# Kitty Items Scripts

> 

Reviewer: Can we add a short description of what this document is all about? It's not obvious when jumping into it without context.

## Version Check

A script that compares the current process's Node version against the [package.json](/package.json) Node engine requirement.

## Startup

This script is called when deploying both local & testnet versions of the Kitty Items application. Multiple processes are executed to initialize both the database as well as an admin service account required for the application to function.

### Begin Process

> The script begins by ending all running processes from previously launched applications.

Reviewer: ```suggestion
The script begins by ending all running processes from previous runs of the script.
```

### Check for correct Node version

> In this step, the script checks the process's Node version against the Node engine requirement in the root [package.json](/package.json). Similar to [version-check.js](/.ki-scripts/version-check.js).

Reviewer: Why is this done twice?

Author: the stand-alone version check script is ran through `npm run preinstall` 
the logic is also in the startup script. adding a line to the `Version Check` description on how to run the preinstall script

### Testnet account creation (`bootstrapNewTestnetAccount()`)

> This section defines the function that begins the process of creating a new Testnet account for applications running on the testnet environment. This account becomes the admin service account that eventually mints your Kitty Items on the testnet.

Reviewer: ```suggestion
This section defines the function that begins the process of creating a new Testnet account for applications that connect to and run Cadence on the testnet. This account becomes the admin service account that eventually mints your Kitty Items on the testnet.
```

Reviewer: I'm still not sure whether to use "the testnet" or Testnet ... but I think we should pick one per doc and stick to it, instead of having both. This is a minor complaint, but it bugs me.

Author: going with Testnet

### Contract deployment (`deployAndInitialize()`)
This section defines the function that begins the process of deploying the required cadence contracts to the admin service account. The contracts being deployed can be found in [flow.json](/flow.json).

- **Initialize Kitty Items**: This step executes the [setup_account.cdc](/cadence/transactions/kittyItems/setup_account.cdc) transaction and initializes the admin service account with the resources required to mint & transfer Kitty Items.

- **Initialize NFTStorefront**: This step executes the [setup_account.cdc](/cadence/transactions/nftStorefront/setup_account.cdc) transaction and initializes the admin service account with the resources required to create Kitty Item listings on the marketplace.

### Emulator environment startup

> This process initializes the Flow emulator and creates a local instance of the Flow blockchain to develop against. The emulator will also initialize an instance of the FCL Dev Wallet. Once the processes are running, the script will execute the `Contract deployment` steps to set up the admin service account.

Reviewer: ```suggestion
This process initializes the Flow emulator which starts a local instance of the Flow blockchain protocol to develop against. The emulator will also initialize an instance of the [FCL Dev Wallet](https://github.com/onflow/fcl-dev-wallet). Once the processes are running, the script will execute the `Contract deployment` steps to set up the admin service account.
```
